libraries/AP_HAL_ChibiOS/hwdef/F4BY_H743/README.md

```diff
@@ -0,0 +1,129 @@
+# F4BY_H743 Flight Controller with STM32H743VIT MCU
+
+The F4BY_H743 flight controller shop: https://f4by.com/en/?order/our_product
+The instructions, schematic, 3D model  are available here: https://f4by.com/en/?doc/fc_f4by_v3.0.1_h743
+
+## Features
+    Processor
+        STM32H743VIT6 MCU (2M flash)
+    Sensors
+        ICM-42688-P IMU (F4BY_H743 v3.0.3)  
+        IIM-42652   IMU (F4BY_H743 v3.0.2)
+        MPU6000     IMU (F4BY_H743 v3.0.1)   
+        MS5611 Barometer 
+        QMC5883P Compass   
+    Power
+        power from external power module (range +5.0 ... +6.0V)
+        anslog current and voltage analog inputs (range 0 ... +6,6V)
+    Interfaces
+        12x PWM outputs DShot capable 
+        1x RC input
+        5x UARTs/serial for GPS, bi-directional RC and other peripherals)
+        1x I2C port for external compass, airspeed, etc.
+        1x CAN bus
+        Buzzer output
+        microSDCard for logging, Lua scripts etc.
+        USB type-C port
+        USB GHS type port for external connector.
+        DFU bootloader button
+
+
+## Analog sensors
+
+- Analog airspeed PC0 (0...+3.3v range) pin 10
+- Analog rssi PC1     (0...+3.3v range) pin 11
+- Voltage sensing PC3 (0...+6.6v range) pin 13
+- Current sensing PC2 (0...+6.6v range) pin 12
+- Board power sensing PC4 (0...+6.6v range)
+- Servo power sensing PC5 (0...+9.9v range)
+
+## UART Mapping
+ 
+- **USB**: SERIAL0
+- **UART1**: SERIAL2  PB6 (TX),  PB7 (RX) – DMA 
+- **USART2**: SERIAL1 PD5 (TX),  PD6 (RX) - DMA at RX   
+- **USART3**: SERIAL3 PD8 (TX),  PD9 (RX) - DMA at RX
+- **UART4**: SERIAL4  PC10 (TX), PC11 (RX)– DMA at RX 
+- **UART5**: Serial5  PC12 (TX), PD2 (RX) – No DMA  
+
+## RC Input
+- RCin  PB0  
+Using the RCin pin will support all unidirectional RC protocols. (PPM, SBUS, iBus, PPM-Sum, DSM,DSM2,DSM-X,SRXL and SUM-D)
+
+- USART2 for Bi-directional protocols (CRSF/ELRS,SRXL2,IRC Ghost, and FPort) see `here <https://ardupilot.org/sub/docs/common-rc-systems.html#common-rc-systems>`
+
+## PWM Output
+
+PWM/DShot capable motor outputs:
```

Comment:
you could set all outputs to be bi-directtional dshot capable given the DMA is not shared